<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Customer</class>
 <widget class="QWidget" name="Customer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>25</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QLabel" name="FirstName_L">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 600 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>first_name</string>
   </property>
  </widget>
  <widget class="QLabel" name="LastName_L">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 600 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>last_name</string>
   </property>
  </widget>
  <widget class="QLabel" name="PhoneNumber_L">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>0</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 600 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>phone_number</string>
   </property>
  </widget>
  <widget class="QLabel" name="Customerid_L">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 600 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>customer_id</string>
   </property>
  </widget>
  <widget class="QLabel" name="Address_TE">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>0</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 600 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>address</string>
   </property>
  </widget>
  <widget class="QLabel" name="Debt_L">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>0</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 600 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>debt</string>
   </property>
  </widget>
  <widget class="QLabel" name="Credit_L">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>0</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 600 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>credit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Edit_PB">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>0</y>
     <width>19</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(6, 106, 110);
    border: none;
    border-radius: 4px;
	padding-left: 2px;
	image: url(:/icons/images/icons/edit.png);
	font: 600 15pt &quot;Segoe UI&quot;;
}
QPushButton:hover {
    background-color: rgb(7, 132, 136);
}
QPushButton::pressed {
    background-color: rgb(5, 97, 100);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Delete_PB">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>0</y>
     <width>19</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(255, 0, 0);
    border: none;
    border-radius: 4px;
	padding-left: 2px;
	image: url(:/icons/images/icons/remove.png);
	font: 600 15pt &quot;Segoe UI&quot;;
}
QPushButton:hover {
    background-color: rgb(7, 132, 136);
}
QPushButton::pressed {
    background-color: rgb(5, 97, 100);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Username_L">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 600 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>username</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>24</y>
     <width>980</width>
     <height>1</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>199</red>
         <green>199</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>199</red>
         <green>199</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>199</red>
         <green>199</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>199</red>
         <green>199</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>199</red>
         <green>199</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>199</red>
         <green>199</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>199</red>
         <green>199</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>199</red>
         <green>199</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>199</red>
         <green>199</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(199, 199, 190);
    height: 0.5px;
    border: none;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>Edit_PB</zorder>
  <zorder>Delete_PB</zorder>
  <zorder>Customerid_L</zorder>
  <zorder>Username_L</zorder>
  <zorder>FirstName_L</zorder>
  <zorder>LastName_L</zorder>
  <zorder>PhoneNumber_L</zorder>
  <zorder>Address_TE</zorder>
  <zorder>Debt_L</zorder>
  <zorder>Credit_L</zorder>
  <zorder>line</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
